app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,8 @@
     <%= javascript_importmap_tags %>
     <%= javascript_include_tag 'room_bookings' %>
   </head>
+  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"></script>
```

Comment:
What do we need the bootstrap js files for?

Reply:
The view preferences button